src/Razor/Microsoft.AspNetCore.Razor.Language/src/DefaultRazorCodeGenerationOptions.cs

```diff
@@ -36,5 +38,7 @@ internal class DefaultRazorCodeGenerationOptions : RazorCodeGenerationOptions
         public override bool SuppressChecksum { get; }
 
         public override bool SuppressNullabilityEnforcement { get; }
+
+        public override bool OmitMinimizedComponentAttributeValues { get; }
```

Comment:
I've defined this new option, instead of using the `RazorLanguageVersion` directly inside the code writer because:

1. It seems like the existing philosophy is that the language version should be used to trigger the values for options, rather than code doing things directly based on the language version, so this is consistent.
2. The code writer has no existing way of knowing the `RazorLanguageVersion` (perhaps because of the philosophy mentioned above), and threading it through many layers would be quite disruptive, including to some public APIs

Reply:
Don't have to address in this PR but might want to add a new `CodeGenerationFeatureFlags` property to contain these similar to [this](https://github.com/dotnet/aspnetcore/blob/66dd4919dafcb21c57a923a661cdbb0ff351713d/src/Razor/Microsoft.AspNetCore.Razor.Language/src/DefaultRazorParserOptions.cs#L22). That makes it cleaner if we introduce more code generation features in the future.